### PR TITLE
Update use-spam-notifications-to-release-and-report-quarantined-messa…

### PR DESCRIPTION
**Pull request by msbemba**
…ges.md

Added a note about the blocked sender and what happens after we block them

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3143

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/use-spam-notifications-to-release-and-report-quarantined-messages.md [hunk 1]
```diff
@@ -53,3 +53,6 @@ An end-user spam notification contains the following information for each quaran
 - **Review**: Click this link to go to Quarantine in the Security & Compliance Center, where you can (depending on why the message was quarantined) view, release, delete or report your quarantined messages. For more information, see [Find and release quarantined messages as a user in EOP](find-and-release-quarantined-messages-as-a-user.md).
 
 ![Example end-user spam notification](../../media/end-user-spam-notification.png)
+
+> [!NOTE]
+> A blocked sender can still send you mail, but if anything from his or her email address makes it to your email account, it’s immediately moved to the Junk Email folder. Future messages from this sender will go to your Junk Email folder or End-User Quarantine. If you would like to delete these messages on arrival and donot want them to be quarentined, Use [Mail flow Rules](https://docs.microsoft.com/exchange/security-and-compliance/mail-flow-rules/mail-flow-rules) to purge them on arrival.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-spam-notifications-to-release-and-report-quarantined-messages.md]:
```suggestion
> A blocked sender can still send you mail, but if anything from his or her email address makes it to your email account, it’s immediately moved to the Junk Email folder. Future messages from this sender will go to your Junk Email folder or to end-user quarantine. If you would like to delete these messages on arrival and don't want them to be quarantined, use [Mail flow Rules](https://docs.microsoft.com/exchange/security-and-compliance/mail-flow-rules/mail-flow-rules) to purge them on arrival.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**